### PR TITLE
Update training curve plot.

### LINKED ISSUE #545: Explain Cross Entropy Loss
Episode 2 gives a high-level explanation of (Categorical) Cross Entropy Loss in section 5.
Perhaps add a box or a link (e.g. to [PyTorch documentation](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html)) that explains this in more depth and discusses alternative loss functions.

### PR DESCRIPTION
**Pull request by carschno**
Fixes #545. 
Only updated the plot because the explanation about Cross Entropy had already been added.

### CODE REVIEW

**Review by angel-daza — APPROVED** [review]
nice that you added the alt-text for the training curve image already
